src/get-started/java-binding.md

```diff
@@ -466,6 +498,7 @@ For using the library just include the dependency in your `pom.xml`:
 [abstractservice]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/service/AbstractService.html
 [apicontrollertest]: https://github.com/exonum/exonum-java-binding/blob/v0.3/exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java
 [app-tutorial]: https://github.com/exonum/exonum-java-binding/blob/master/exonum-java-binding-core/rust/ejb-app/TUTORIAL.md
+[blockchain]: https://github.com/exonum/exonum-java-binding/0.4/master/exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/Blockchain.java
```

Comment:
link is broken

Reply:
I'd add any link to make the linter happy and a todo.

Reply:
Sure, I wasn't sure what link to put it - as this doc will be released after 0.4, the link should be valid after that.
Maybe it's better to put a Javadoc link there, instead of a source code one?